src/global-settings.php

```diff
@@ -809,6 +863,121 @@ public function posts_block_columns_fix( $selectors, $tag ) {
 
 			return $selectors;
 		}
+
+		/**-----------------------------------------------------------------------------
+		 * Block Layouts functions
+		 *-----------------------------------------------------------------------------*/
+		/**
+		 * Generate CSS for:
+		 *  - Global Spacing and Borders
+		 *  - Global Buttons and Icons
+		 *
+		 * @param String $current_css
+		 * @return String
+		 */
+		public static function generate_global_block_layouts( $option_name, $settings_name, $defaults ) {
+			$block_layouts = get_option( $option_name );
+
+			if ( ! $block_layouts || ! is_array( $block_layouts ) ) {
+				return false;
+			}
+
+			$tablet_breakpoint = 1023;
+			$mobile_breakpoint = 767;
```

Comment:
We should be using the same breakpoints as our other breakpoints right?
```suggestion
			$tablet_breakpoint = 1025;
			$mobile_breakpoint = 768;
```